app/models/king.rb

```diff
@@ -1,2 +1,16 @@
 class King < Piece
+  def valid_move?(_destination_row, _destination_col)
+    kings_direction = [
+      [1, 0],
+      [-1, 0],
+      [0, -1],
+      [0, 1],
+      [1, -1],
+      [1, 1],
+      [-1, -1],
+      [-1, 1]
+    ]
+    return true if kings_direction.include?(direction)
```

Comment:
Ahhh. I left out the direction code line   I think I deleted that line when I refactored